pkg/server/server.go

```diff
@@ -236,11 +249,21 @@ func New(
 		}
 
 		if v := tuning.Snapshot; v != nil {
-			logrus.WithFields(logrus.Fields{"threshold": v.Threshold, "trailing": v.Trailing}).Print("Configure dqlite raft snapshot parameters")
-			options = append(options, app.WithSnapshotParams(dqlite.SnapshotParams{
-				Threshold: v.Threshold,
-				Trailing:  v.Trailing,
-			}))
+			logrus.WithFields(logrus.Fields{"threshold": v.Threshold, "trailing": v.Trailing}).Print("Initial dqlite raft snapshot parameters before adjustments")
+
+			if v.Trailing < minTrailing && v.Threshold > minThreshold {
+				snapshotParameters.Trailing = v.Threshold * scalingFactor
```

Comment:
Should this also change to `v.Trailing`?
```suggestion
				snapshotParameters.Trailing = v.Trailing * scalingFactor
```

Reply:
The relationship for trailing:threshold is 1:2, so if one of the given parameters that the user gives us is to small we'll compute it from the other value.